### PR TITLE
Events Page

### LINKED ISSUE #1: Events page
~Make a Gantt Chart~
- [ ] Display Events using Hover Card
- [ ] Create Induvial Page for each event

### PR DESCRIPTION
**Pull request by rabilrbl**
resolves #1

### CODE REVIEW

File: src/app/api/gallery/route.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export async function GET() {
   const imageDirectory = path.join(process.cwd(), "public", "gallery");
   const imageFiles = fs.readdirSync(imageDirectory);
   const images = imageFiles.map((file: string, i: number) => ({
-    alt: `Image ${i}`,
+    alt: `Imagae ${i}`,
```
Comment (anishshobithps) [inline src/app/api/gallery/route.ts:8]: Should this be changed ?

File: src/components/ui/timechart.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import EventCard from "./EventCard";
 
 const timechart = () => {
```
Comment (anishshobithps) [inline src/components/ui/timechart.tsx]: as we are not using timechart, let's not make it timechart or name it that way, let's give proper name for the same ? Also I would prefer Named exports over default

**Review by anishshobithps — CHANGES_REQUESTED** [review]

**Comment by anishshobithps** [comment]
@melroydsa there seems to be a issue, 

```
<html>
<body>
<!--StartFragment-->
Failed to compile.
--
15:24:14.161 |  
15:24:14.161 | ./src/components/ui/timechart.tsx:2:8
15:24:14.161 | Type error: Module '"/vercel/path0/src/components/ui/EventCard"' has no default export. Did you mean to use 'import { EventCard } from "/vercel/path0/src/components/ui/EventCard"' instead?
15:24:14.161 |  
15:24:14.161 | 1 \| import React from "react";
15:24:14.161 | > 2 \| import EventCard from "./EventCard";
15:24:14.161 | \|        ^
15:24:14.161 | 3 \|
15:24:14.161 | 4 \| const timechart = () => {
15:24:14.161 | 5 \|   const events = [
15:24:14.205 | Error: Command "npx prisma generate && next build" exited with 1

<!--EndFragment-->
</body>
</html>
```

**Comment by anishshobithps** [comment]
I suppose the background beam is heavy, its better if its off, on deployment its lagging for me

**Comment by anishshobithps** [comment]
> I suppose the background beam is heavy, its better if its off, on deployment its lagging for me

@melroydsa can you remove the background beams ? its lagging

**Comment by melroydsa** [comment]
Alright

**Review by anishshobithps — APPROVED** [review]

**Pull request merged**